bodybank/character_generator/admin.py

```diff
@@ -1,36 +1,59 @@
 from django.contrib import admin
-from .models import Ego, Movement, Morph, Background, Skill, Background2Skills, Career, Career2Skills
+from .models import (
+    Ego,
+    Movement,
+    Morph,
+    Background,
+    Skill,
+    Background2Skills,
+    Career,
+    Career2Skills,
+    Item,
+)
 
 
 class EgoAdmin(admin.ModelAdmin):
     pass
 
+
 class MovementAdmin(admin.ModelAdmin):
     pass
 
+
 class MorphAdmin(admin.ModelAdmin):
     pass
 
+
 class BackgroundAdmin(admin.ModelAdmin):
     pass
 
+
 class SkillAdmin(admin.ModelAdmin):
     pass
 
+
 class Background2SkillsAdmin(admin.ModelAdmin):
     pass
 
+
 class CareerAdmin(admin.ModelAdmin):
     pass
 
+
 class Career2SkillsAdmin(admin.ModelAdmin):
     pass
 
+
+@admin.register(Item)
+class ItemAdmin(admin.ModelAdmin):
+    list_display = ("name", "itype", "complexity_gp", "desc")
```

Comment:
More cool things you can do, but list, search and filter are the most common ones:
https://docs.djangoproject.com/en/4.1/ref/contrib/admin/

For bookmark / later reading: 
https://books.agiliq.com/projects/django-admin-cookbook/en/latest/
-> some really cool tricks